### PR TITLE
blacklist object and $UID exposed from Foam

### PR DESCRIPTION
**Pull request by jingt06**

### CODE REVIEW

File: src/ObjectGraph.es6.js [hunk 1]
```diff
@@ -62,15 +62,17 @@ ObjectGraph.prototype.types = ObjectGraph.types = {
 };
 
 // Never visit/store these object keys.
-ObjectGraph.prototype.blacklistedKeys = [ uid.key, uid.key__, '__proto__' ];
+ObjectGraph.prototype.blacklistedKeys = [ uid.key, uid.key__, '__proto__',
+  '$UID', '$UID__']; // Foam assigns unique Ids to objects.
```
Comment (mdittmer) [inline src/ObjectGraph.es6.js]: Please clarify that this refers to the last two keys in the list. Also, please add a TODO to upstream id-js to FOAM so we don't have two unique identifiers floating around.

Comment (mdittmer) [reply src/ObjectGraph.es6.js]: Nit: here and below, capitalization is "FOAM Framework"

**Review by mdittmer — CHANGES_REQUESTED** [review]

File: src/ObjectGraph.es6.js (hunk 1 above)
Comment (jingt06) [reply src/ObjectGraph.es6.js]: Fixed

File: src/ObjectGraph.es6.js [hunk 2]
```diff
@@ -62,8 +62,12 @@ ObjectGraph.prototype.types = ObjectGraph.types = {
...
 
 // Never visit/store these object keys.
-ObjectGraph.prototype.blacklistedKeys = [ uid.key, uid.key__, '__proto__',
-  '$UID', '$UID__']; // Foam assigns unique Ids to objects.
+ObjectGraph.prototype.blacklistedKeys = [
+  uid.key,
+  uid.key__,
+  '__proto__',
+  '$UID',    // $UID and $UID__ are unique ids assigned by FOAM Framework.
+  '$UID__']; // TODO: Upstream id-js to FOAM so we don't have two unique ids.
```
Comment (mdittmer) [inline src/ObjectGraph.es6.js]: nit: Please add trailing comma and close array literal on its own line.

**Review by mdittmer — APPROVED** [review]
Please fix nit, then LGTM!

**Pull request merged**